tests/create_LWE.py

```diff
@@ -0,0 +1,231 @@
+from addict import Dict  
+
+from CADETProcess.simulator import Cadet
+
+from CADETProcess.processModel import (
+
+    ComponentSystem,
+    FlowSheet,
+
+    Inlet,
+    Outlet,
+    Cstr,
+    GeneralRateModel,
+    TubularReactor,
+    LumpedRateModelWithoutPores,
+    LumpedRateModelWithPores,
+    MCT,
+
+    Linear,
+    Langmuir,
+    LangmuirLDF,
+    BiLangmuir,
+    BiLangmuirLDF,
+    FreundlichLDF,
+    StericMassAction,
+    AntiLangmuir,
+    Spreading,
+    SelfAssociation,
+    BiStericMassAction,
+    MultistateStericMassAction,
+    SimplifiedMultistateStericMassAction,
+    Saska,
+    GeneralizedIonExchange
+)
+
+
+def create_lwe(options: dict) -> None:
+
+    options = Dict(options)  
+
+    component_system = ComponentSystem(options.n_comp)
+    flow_sheet = FlowSheet(component_system)
+
+    unit_type = options.unitType
+    if unit_type == "cstr":
+        configure_cstr(component_system, options)
+    elif unit_type == "generalratemodel":
+        configure_grm(component_system, options)
+    elif unit_type == "tubularreactor":
+        configure_tb(component_system, options)
+    elif unit_type == "lumpedratemodelwithoutpores":
+        configure_lrm(component_system, options)
+    elif unit_type == "lumpedratemodelwithpores":
+        configure_lrmp(component_system, options)
+    elif unit_type == "mct":
+        configure_mct(component_system, options)
+    else:
+        raise ValueError(f"Unknown unit operation type {unit_type}")
+
+def configure_cstr(component_system, options):
+
+    cstr = Cstr(component_system, name='cstr')
+    cstr.V = 1e-3
+    cstr.porosity = 0.37 + (1.0 - 0.37) * 0.75
+
+    configure_adsorption(options.n_partype, options.is_kinetic)
+
+def configure_grm(component_system, options):
```

Comment:
[suggestion] Consider being more explicit with options on the lower levels of the configure functions, i.e. instead of a dictionary, add the options as kwargs.

```suggestion
def configure_grm(component_system, discretization_parameters, adsorption_parameters, reverse_flow):
```

Reply:
I solved this one a bit differently, please have a look and let's decide how explicit we want to be on the lower levels.